packages/components/psammead-episode-list/src/helpers.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+// Used to make service, script and dir passed to <EpisodeList> available to children
```

Comment:
```suggestion
// Used to make props passed to <EpisodeList> available to children
```
Current limitation here where the `Episode` component cannot be used in isolation - can only be used as a child of `EpisodeList` due to the way context is implemented - could be resolved later